boards/rp-pico-w/examples/pico_w_blinky.rs

```diff
@@ -0,0 +1,83 @@
+//! # Pico Blinky Example
+//!
+//! Blinks the LED on a Pico board.
+//!
+//! This will blink an LED attached to GP25, which is the pin the Pico uses for
+//! the on-board LED.
```

Comment:
```suggestion
//! # Pico W Blinky Example
//!
//! Blinks the LED on a Pico W board.
//!
//! This will blink an LED attached to WL_GPIO0, which is the pin the Pico W uses for
//! the on-board LED. It is connected to the the Wifi chip so it cannot be set using RP2040 pins
```